snapcraft.yaml

```diff
@@ -1356,7 +1358,7 @@ parts:
     organize:
       lib/python3.12/site-packages/: lib/python3/dist-packages/
     prime:
-      - bin/uefivars.py
+      - bin/uefivars
```

Comment:
will this break lxd's integration?

Reply:
Yes it will, thanks for catching this! I don't have time today to work on a fix in LXD, though.